eclcli/dns/v2/record.py

```diff
@@ -153,9 +153,14 @@ def get_parser(self, prog_name):
             help="ID of the zone which recordset you want to show belong to.",
         )
         parser.add_argument(
-            "recordset_id",
-            metavar="<record_id>",
-            help="ID of the recordset you want to show.",
+            "recordset_id_or_name",
```

Comment:
name is not unique in records and show command must return single record. That's why we cannot use name instead of id.